components/suggestion/switch_channel_provider.jsx

```diff
@@ -142,15 +142,17 @@ class SwitchChannelSuggestion extends Suggestion {
                 onClick={this.handleClick}
                 onMouseMove={this.handleMouseMove}
                 className={className}
+                role='list'
```

Comment:
Perhaps this should be the `listitem` instead of the span within it? Does that make this get read properly?

Reply:
Otherwise it doesn't read the other channels

Reply:
Maybe `role='list'` should be set on the `SuggestionList` instead since we're not using a proper list element?